src/main/scala/firrtl/Compiler.scala

```diff
@@ -113,6 +113,7 @@ trait SimpleRun extends LazyLogging {
   def runPasses(circuit: Circuit, passSeq: Seq[Pass]): Circuit =
     passSeq.foldLeft(circuit) { (c: Circuit, pass: Pass) =>
       val x = Utils.time(pass.name) { pass.run(c) }
+      logger.debug(s"** Pass ${pass.name} **")
```

Comment:
So this actually **should** be unnecessary. Utils.time prints the name and timing of the pass in info, and info **should** print when debug is enabled but I think there's a bug in the logger.